src/compat/text.rs

```diff
@@ -161,12 +161,14 @@ where
     for line in reader.lines() {
         let line =
             line.map_err(|e| ErrorKind::io_error("Cannot read line from embedding file", e))?;
-        let mut parts = line.split_whitespace();
+        let mut parts = line
+            .split(|c| c == ' ' || c == '\t')
+            .filter(|c| !c.is_empty());
```

Comment:
Maybe rename `c` here, because the elements are not characters anymore?

Also, maybe you could put a comment here that once we have 1.34.0 as a minimum, this can be replaced by `split_ascii_whitespace`?

https://doc.rust-lang.org/std/primitive.str.html#method.split_ascii_whitespace

Reply:
> Maybe rename c here, because the elements are not characters anymore?

Yeah, `part` makes more sense.

> Also, maybe you could put a comment here that once we have 1.34.0 as a minimum, this can be replaced by split_ascii_whitespace?

Weird, that works. I was pretty sure that I tried that yesterday. Probably hadn't yet realized that I was trimming words into nothingness.